climada/hazard/tc_tracks_forecast.py

```diff
@@ -56,7 +56,8 @@
     'A': 'A - Arabian Sea (North Indian Ocean)',
     'B': 'B - Bay of Bengal (North Indian Ocean)',
     'U': 'U - Australia',
-    'S': 'S - South-West Indian Ocean'
+    'S': 'S - South-West Indian Ocean',
+    'X': 'X - Undefined Basin'
 }
 """Gleaned from the ECMWF wiki at
 https://confluence.ecmwf.int/display/FCST/Tropical+Cyclone+tracks+in+BUFR+-+including+genesis
```

Comment:
I think it's worth extending this comment to explain where 'X' comes from

```suggestion
https://confluence.ecmwf.int/display/FCST/Tropical+Cyclone+tracks+in+BUFR+-+including+genesis
with added basin 'X' to deal with it appearing in operational forecasts
(see e.g. years 2020 and 2021 in the sidebar at https://www.ecmwf.int/en/forecasts/charts/tcyclone/)
```